### DISCUSSION TITLE
Unable to type in command prompt on windows

### DISCUSSION THREAD

**Discussion by Zeprix** [dyc3/steamguard-cli#387] [Q&A]
### Version

steamguard-cli 0.14.0

### What happened?

If I run the program on PowerShell, CMD or run the .exe directly, but I'm unable to type anything once the program is initialized. I tried setting up a new account, but I count input the username.

**Comment by dyc3** [comment]
Make sure you aren't accidentally selecting any characters when you focus the window to type in it. IIRC that prevents you from typing.

**Reply by Zeprix** [reply]
> Make sure you aren't accidentally selecting any characters when you focus the window to type in it. IIRC that prevents you from typing.

Apparently the issue isnt that the text isnt being written, it is that it isnt displayed. When I hit enter it works.

**Reply by dyc3** [reply]
Your password is intentionally not displayed as you type it. It's a security feature.

**Reply by Zeprix** [reply]
> Your password is intentionally not displayed as you type it. It's a security feature.

I have the same issue with my username, verification codes and etc.

**Reply by dyc3** [reply]
Hmm, yeah that's definitely not supposed to happen. I'll look into it.
